modules/drivers/oracle/test/metabase/driver/oracle_test.clj

```diff
@@ -231,3 +233,13 @@
                                                   &test_data_categories__via__cat.categories.id]
                                    :fk-field-id  (mt/id :venues :category_id)
                                    :fields       :none}]}))))))))
+
+(deftest oracle-connect-with-ssl-test
+  (mt/test-driver :oracle
+    (if (System/getenv "MB_ORACLE_SSL_TEST_SSL")
+      (testing "Oracle with SSL connectivity"
+        (tu/with-env-keys-renamed-by #(str/replace-first % "mb-oracle-ssl-test" "mb-oracle-test")
+          (metabase.query-processor-test.order-by-test/order-by-aggregate-fields-test)))
```

Comment:
I would generally avoid using the full namespace like this; here's my reasoning:

Using something from a namespace without requiring it creates a race condition in the namespace that uses it. e.g. suppose I have a namespace like

```clj
(ns my-namespace)

(defn my-fn [x]
  (clojure.pprint/pprint x))
```

`(require 'my-namespace)` will work fine 99% of the time, so long as `clojure.pprint` has already been loaded by someone else. However, if you attempt to load `my-namespace` before `clojure.pprint` has been loaded, it will fail because `clojure.pprint` doesn't exist (yet). So you have a race condition when loading this namespace, as it's implicitly dependent on someone else loading things it uses before it gets loaded.

So any time you see a symbol qualified by the entire namespace like that, you can't be sure it's not a situation where it's used without being required _unless_ you jump to the `ns` form to double-check. On the other hand, if you use an alias of some sort, anyone can be sure you've properly required the namespace without having to jump to the `ns` form to look.

I know you *did* require this namespace, but it's nicer if other people won't have to double-check this in the future

Reply:
Thanks for the explanation.  Fixed now.